src/logic/createStyle.ts

```diff
@@ -1,4 +1,26 @@
 import { Keyframes } from '../types';
+import camelCaseToDash from '../utils/camelCaseToDash';
+
+const generateKeyframes = keyframes => {
+  const animationLength = keyframes.length;
+  return keyframes.reduce((previous, keyframe, currentIndex) => {
+    const keyframePersentage = animationLength === 2
+      ? currentIndex * 100
+      : parseFloat((100 / (animationLength - 1)).toFixed(2)) * currentIndex;
+      // ['opacity: 0', 'opacity: 1']
+      if (typeof keyframe === 'string') {
+        return `${previous} ${keyframePersentage}% {${keyframe}}`;
+      }
+      const keys = Object.keys(keyframe);
+      // [{opacity: 0}, {opacity: 1}]
+      if (keys.length && isNaN(+keys[0])) {
```

Comment:
do you need this `+`?

Reply:
@bluebill1049 I don't but don't know how to pypass the flow since isNaN only takes numbers... Any suggestion?

Reply:
nah all good, leave it bro @pangpang1987

Reply:
@bluebill1049 I am new to ts. Just find out how to ignore the error...